db/migrate_cards/20170309143754_award_badges.rb

```diff
@@ -0,0 +1,155 @@
+# -*- encoding : utf-8 -*-
+
+class AwardBadges < Card::Migration
+  #disable_ddl_transaction!
+
+  def up
+    # award_badges_by_user
+    # award_answer_create_badges
+    # [:project
+    # #, :source, :metric, :wikirate_company
+    # ].each do |type_code|
+    #   award_create_badges type_code
+    # end
+    # binding.pry
+    # card = Card.fetch "some random card", new: {}
+    # card.save!
+    #card = Card.fetch "Philipp Kuehl+Project+Badges earned", new: { type_id: Card::PointerID }
+    #card.add_badge_card Card.fetch("Project Launcher")
+    #card.save!
```

Comment:
I moved it to a script because the integration stage is not executed in migrations. I added a PT story for that.